src/AppBundle/Entity/Delivery.php

```diff
@@ -45,6 +45,8 @@ class Delivery extends TaskCollection implements TaskCollectionInterface
     const VEHICLE_BIKE = 'bike';
     const VEHICLE_CARGO_BIKE = 'cargo_bike';
 
+    const COLORS_LIST = ['#213ab2', '#b2213a', '#5221b2', '#93c63f', '#b22182', '#3ab221', '#b25221', '#2182b2', '#3ab221', '#9c21b2', '#c63f4f', '#b2217f', '#82b221', '#5421b2', '#3f93c6', '#21b252', '#c6733f'];
```

Comment:
This is too much coupling between the view & the model. 
This should be done in Twig / JavaScript side.

Reply:
Advantage to having it here is that it is easy to serialize `color` in the API (so we have exactly same colors everywhere).

Maybe having a service that get the color is a better solution?